game/client/sdRenderer.js

```diff
@@ -273,12 +273,7 @@ class sdRenderer
 					//if ( typeof image_obj_cache[ complex_filter_name ] === 'undefined' )
 					if ( !image_obj_cache_named_item )
 					{
-						if ( typeof OffscreenCanvas !== 'undefined' )
-						var ctx_check = new OffscreenCanvas( 32, 32 ).getContext("2d").filter;
-						else
-						var ctx_check = undefined;
-
-						if ( typeof OffscreenCanvas !== 'undefined' && ctx_check !== undefined )
+						if ( typeof OffscreenCanvas !== 'undefined' && "filter" in OffscreenCanvasRenderingContext2D.prototype )
```

Comment:
I have feeling like this will crash if there is no OffscreenCanvasRenderingContext2D global object.